Homework 9/Ex6/Program.cs

```diff
@@ -0,0 +1,92 @@
+﻿class Company
+{
+    public string Name { get; set; }
+    public int Id { get; set; }
+    public int NumberOfEmployees { get; set; }
+}
+
+class CompanyService
```

Comment:
Мы пока не разбирали отдельно тему наследования, но данный класс можно было бы улучшить, добавив наследование класса `CompanyService` от `IEnumerable<Company>`:
```suggestion
class CompanyService : IEnumerable<Company>
```
Это сделает код более гибким, удобным и поддерживаемым. Ты сможешь использовать стандартные механизмы перебора коллекций, такие как цикл `foreach`, без необходимости вручную реализовывать метод `GetEnumerator`. `IEnumerable<T>` - это стандартный интерфейс коллекций в .NET, который совместим с различными библиотеками. Тебе это позволит использовать LINQ-запросы (о которых мы с тобой отдельно поговорим на одном из следующий занятий) и другие стандартные методы для работы с коллекцией.